idevice/src/services/os_trace_relay.rs

```diff
@@ -61,24 +61,24 @@ impl IdeviceService for OsTraceRelayClient {
 
 /// An initialized client for receiving logs
 pub struct OsTraceRelayReceiver {
-    inner: OsTraceRelayClient,
+    pub inner: OsTraceRelayClient,
```

Comment:
Is there a reason to make this pub? It's unusable after starting the relay.

Reply:
Oh it was probably from me experimenting with the bindings, will remove!